astroid/cache.py

```diff
@@ -0,0 +1,100 @@
+""" Caching utilities used in various places."""
+
+from collections import OrderedDict
+from weakref import WeakSet
+
+import wrapt
+
+LRU_CACHE_CAPACITY = 128
```

Comment:
Do we want to this to be set-able somehow? I can imagine instances where somebody would want to cache to be larger?

Reply:
@DanielNoord I'm not sure what you mean by "set-able"; it is settable. Would you like to add a setter method? Or make it overridable by instances?

Reply:
We probably need to document the variable and explains explicitely how to modify it (assign a new value to it, I suppose). We can add the option in pylint config later but astroid users will need the info.

Reply:
Preferably we could change this setting via an option, but now that I think of it I don't think we really have a good way to do so.

I'll create a "create option-setting system" project after this gets merged and add this global to the "to be configurable" list.